src/auxiliary.cpp

```diff
@@ -2,10 +2,10 @@
 #include "openfhe.h"
 
 void wrap_auxiliary(jlcxx::Module& mod) {
-  // Helper function for getting the text representation where only operator<< is defined
+// Helper function for getting the text representation where only operator<< is defined
```

Comment:
```suggestion
  // Helper function for getting the text representation where only operator<< is defined
```
I think the indent was correct before?